packages/flame_texturepacker/example/assets/FlameAtlasMap.atlas

```diff
@@ -0,0 +1,102 @@
+
```

Comment:
for simplicity, we should use the same file name in the example and in the readme snippet

Reply:
Fixed!